packages/integrations/deno/src/index.ts

```diff
@@ -149,9 +150,6 @@ export default function createIntegration(args?: Options): AstroIntegration {
 							(vite.resolve.alias as Record<string, string>)[alias.find] = alias.replacement;
 						}
 					}
-					vite.ssr = {
-						noExternal: COMPATIBLE_NODE_MODULES,
-					};
```

Comment:
`noExternal` was set to true originally. I had to change it to the list of bare node built-in modules in #7288. But apparently, it doesn't need to be set at all.